EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -981,6 +982,12 @@ unsigned int V2::World::countCivilizedNations() const
 
 void V2::World::convertProvinces(const EU4::World& sourceWorld, const mappers::TechGroupsMapper& techGroupsMapper, const EU4::Regions& eu4Regions)
 {
+	std::optional<std::string> v2HreTag;
+	if (const auto& eu4HreTag = sourceWorld.getShatteredHreTag(); eu4HreTag)
```

Comment:
move this above to where you load it.

Reply:
> move this above to where you load it.

Just to make sure, this stays where it is, because of that deleting country issue?

Reply:
correct, leave as is.